flink/src/test/java/org/apache/iceberg/flink/FlinkTestBase.java

```diff
@@ -34,8 +37,27 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.rules.TemporaryFolder;
 
-public abstract class FlinkTestBase extends AbstractTestBase {
+public abstract class FlinkTestBase extends TestBaseUtils {
+
+  private static final int DEFAULT_PARALLELISM = 4;
+
+  private static final Configuration config = new Configuration()
+          // disable classloader check as Avro may cache class/object in the serializers.
+          .set(CoreOptions.CHECK_LEAKED_CLASSLOADER, false);
```

Comment:
I think we need to disable the `CoreOptions.CHECK_LEAKED_CLASSLOADER` here  ?  Seems like we usually validation the iceberg tables results once we've terminated the flink job.

Reply:
It is already disabled here. We need to disable this check for other tests that are affected by this.

Reply:
This is getting really tricky to debug. Can't reproduce it locally. Hence, can't set up break point or add debug logging to custom built Flink.

I tried to disable `CoreOptions.CHECK_LEAKED_CLASSLOADER` for all MiniClulster tests. That doesn't help.

Reply:
OK, I'd see how to reproduce those class loader issues.

Reply:
There're still some ITCases which did not disable the `CHECK_LEAKED_CLASSLOADER` in current patch, so I made this commit: https://github.com/openinx/incubator-iceberg/commit/c8c2c8e8e72fd1b2051f1f946bb54c968e86c81d to re-trigger the travis CI again, let's see what will it say: https://github.com/openinx/incubator-iceberg/runs/1806279011

Reply:
If the travis CI pass, then @stevenzwu, you can apply this commit in this patch to retry this travis CI: 

```
curl -s https://github.com/openinx/incubator-iceberg/commit/c8c2c8e8e72fd1b2051f1f946bb54c968e86c81d.patch | git am
```

Reply:
Seems all unit tests are passed https://github.com/openinx/incubator-iceberg/runs/1806279011 ! @stevenzwu , you could add this commit and then retry the travis CI again.

Reply:
@openinx thx a lot for identifying the missing integration tests that require disabling classloader check. I didn't realize the need to search for `StreamExecutionEnvironment.getExecutionEnvironment`